### ISSUE TITLE
[JavaScript] Scopes brake on class methods defined using square brackets

### ISSUE THREAD

**Issue by rchl** [sublimehq/Packages#554]
Sublime 3 build 3119.

Scopes are broken after opening parenthesis.
```
class Foo {
  [SOME_CONSTANT](arg) {
    return this.a;
  }
}
```

**Comment by rchl** [comment]
Similar example from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes:
```
class MyArray extends Array {
  // Overwrite species to the parent Array constructor
  static get [Symbol.species]() { return Array; }
}
```

**Comment by wbond** [comment]
This should be resolved by 6618afe86c81ef3c9efb74b08376159bb2642b2b
